TUnit.Assertions/AssertConditions/Connectors/OrAssertCondition.cs

```diff
@@ -12,18 +12,23 @@ public OrAssertCondition(BaseAssertCondition<TActual> condition1, BaseAssertCond
         
         _condition1 = condition1;
         _condition2 = condition2;
-    }
+	}
 
-    protected internal override string GetFailureMessage() => $"{_condition1.OverriddenMessage ?? _condition1.GetFullFailureMessage()}{Environment.NewLine} or{Environment.NewLine}{_condition2.OverriddenMessage ?? _condition2.GetFullFailureMessage()}";
+    protected override string GetExpectation()
```

Comment:
Does this not get called? If not I'd prefer to return an empty string than throw not supported.

Reply:
No, it does not get called, as I overwrite the `GetExpectationWithReason` and combine the message from `_condition1` and `_condition2`.
I will change it to return an empty string and add a comment.